var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -250,6 +250,6 @@ def write_rpath_specs(self):
             for line in lines:
                 out.write(line + '\n')
                 if line.startswith('*link:'):
-                    out.write(r'-rpath {0}:{1} \n'.format(
```

Comment:
Oh, I see! So `'\\\n'` looks like `\` with a newline character after it and `r'\n'` just looks like `\n`. That actually makes sense. Thanks for teaching me about raw strings @hartzell!